generators/src/new-test-model/new-test-model.generator.ts

```diff
@@ -188,6 +195,61 @@ export const newTestModelGenerator: CodeGenerator = {
         });
     }
 
+    // 3. Update the single package preset
+    if (generationType === 'standalone') {
+      const presetDirectoryPath = join(
+        __dirname,
+        '..',
+        '..',
+        '..',
+        'presets',
+        'all-packages'
+      );
+      // Generate the re-export proxy file
+      writeFileSync(
+        join(presetDirectoryPath, 'src', `${modelCodename}.ts`),
+        `export * from '@commercetools-test-data/${modelCodename}';`
+      );
+      // Update package.json
+      const presetPackageJson = (
+        await import(join(presetDirectoryPath, 'package.json'))
+      ).default;
+      presetPackageJson.files.push(modelCodename);
```

Comment:
maybe worth to add some guards here if the structure isn't expected?

Reply:
Not sure if I follow.
Are you worried about the structure of the `package.json` file import?
If that's the case, the file is managed by us and the `files` property is mandatory. Without it the preset would not work so I think it's safe to assume it will exists.

Please let me know if I misunderstood your comment 🙏